src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/TextDocumentUriPresentationEndpoint.cs

```diff
@@ -165,7 +165,9 @@ protected override IRazorPresentationParams CreateRazorRequestParameters(UriPres
 
         // TODO: Add @using statements if required, or fully qualify (GetTypeName())
 
-        var sb = new StringBuilder();
+        using var pooledBuilder = StringBuilderPool.GetPooledObject();
+        var sb = pooledBuilder.Object;
```

Comment:
Since we've got our own copy of these things, can we fix the API? The newer roslyn style for pooled objects would be to call `using var _ = StringBuilderPool.GetPooledObject(out var builder);` which is much easier to use